chains.py

```diff
@@ -33,10 +33,18 @@ def load_embedding_model(embedding_model_name: str, logger=BaseLogger(), config=
         )
         dimension = 4096
         logger.info("Embedding: Using Ollama")
-    elif embedding_model_name == "openai":
```

Comment:
we should not break the existing config, but leave the existing embeddings and add these new ones explicitely